### PR TITLE
Renames some plotting functions to use _ between words, more in line

### PR DESCRIPTION
**Pull request by bnaecker**
with other parts of the API. Adds a ``to_html()`` function to the
visualizations module, which converts an animation from the ``play_*``
functions to an embeddable HTML object. Adds test for that. Updates the
setup.py file to include the jupyter meta-package as an optional
dependency. Should be able to install this now (once it's on PyPI) with
`pip install "pyret[html]"` if users want the HTML bit.

### CODE REVIEW

File: pyret/visualizations.py [hunk 1]
```diff
@@ -581,3 +581,30 @@ def animate(t):
             np.arange(T), interval=frametime, repeat=repeat)
     return anim
 
+def to_html(anim):
+    """
+    Convert an animation into an embedable HTML element.
+    
+    This converts the animation objects returned by ``playsta()`` and
```
Comment (nirum) [inline pyret/visualizations.py]: play_sta

File: pyret/visualizations.py [hunk 2]
```diff
@@ -581,3 +581,30 @@ def animate(t):
...
+    ``playrates()`` into an HTML tag that can be embedded, for example
```
Comment (nirum) [inline pyret/visualizations.py]: play_rates

File: tests/test_visualizations.py [hunk 3]
```diff
@@ -193,3 +193,18 @@ def test_playrates():
...
+        from IPython.display import HTML
+    except ImportError:
+        pytest.skip('Cannot convert movie to HTML without IPython.')
+
+    nx, ny, nt = 10, 10, 50
+    sta = utils.create_spatiotemporal_filter(nx, ny, nt)[-1]
+    anim = viz.play_sta(sta)
+    html = viz.to_html(anim)
+    assert isinstance(html, HTML)
+
```
Comment (nirum) [inline tests/test_visualizations.py]: remove the empty trailing line at the end of the file?

Comment (bnaecker) [inline tests/test_visualizations.py]: Will do. I'll also add a `v0.5` file to changelogs. If you're on board, we can bump the version to 0.5 when merging this?

**Pull request merged**